build.cmd

```diff
@@ -193,6 +195,21 @@ if %__BuildTypeRelease%==1  set __BuildType=Release
 
 set __RunArgs=-BuildOS=%__BuildOS% -BuildType=%__BuildType% -BuildArch=%__BuildArch%
 
+if %__EnforcePgo%==1 (
+    if %__BuildArchArm%==1 (
+        echo Error: enforcepgo cannot be used with arm architecture
+        goto Usage
+    )
+    if %__BuildArchArm64%==1 (
+        echo Error: enforcepgo cannot be used with arm64 architecture
+        goto Usage
+    )
+    if NOT %__BuildOS% == Windows_NT (
```

Comment:
I added it with the thought that someone may build for Linux from Windows (like we used to do for tests). Is that not important for this case?